app/assets/javascripts/components/onboarding/supplementary.jsx

```diff
@@ -5,6 +5,12 @@ import { connect } from 'react-redux';
 import OnboardAPI from '../../utils/onboarding_utils.js';
 import { addNotification } from '../../actions/notification_actions.js';
 
+const NoLabelStyle = {
+  display: 'inline',
+  fontWeight: 'normal',
+  color: '#6a6a6a'
```

Comment:
Because these styles are used for a very specific use case. Here, its used to override default styles inherited from the form > label CSS. If you want I can add it as a CSS class but it will be hardly used again.

Reply:
I would prefer this to be done in CSS here.